internal/providers/terraform/aws/dynamodb_table.go

```diff
@@ -0,0 +1,117 @@
+package aws
+
+import (
+	"infracost/pkg/schema"
+
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func NewDynamoDBTable(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+	costComponents := make([]*schema.CostComponent, 0)
+	subResources := make([]*schema.Resource, 0)
+
+	// Check billing mode
+	billingMode := d.Get("billing_mode").String()
+	if billingMode != "PROVISIONED" {
+		return nil
```

Comment:
We can now warn for these cases (example: https://github.com/infracost/infracost/blob/refactor/internal/providers/terraform/aws/ecs_service.go#L16)

Reply:
Nice, fixed